mmcls/datasets/samplers/builder.py

```diff
@@ -0,0 +1,10 @@
+from mmcv.utils import Registry, build_from_cfg
```

Comment:
Just place the `build_sampler` and `SAMPLERS` in `mmcls/datasets/builder.py`, instead of a new file.